web/backend/src/Server.ts

```diff
@@ -259,8 +266,8 @@ app.get('/api/user_rights', (req, res) => {
 
 // This call (only for admins) allow an admin to add a role to a voter.
 app.post('/api/add_role', (req, res, next) => {
-  if (!isAuthorized(req.session.userid, SUBJECT_ROLES, ACTION_ADD)) {
-    res.status(400).send('Unauthorized - only admins allowed');
+  if (!isAuthorized(req.session.userid, PERMISSIONS.SUBJECTS.ROLES, PERMISSIONS.ACTIONS.ADD)) {
+    res.status(400).send({ error: 'user not logged-in or does not have enough permissions' });
```

Comment:
why return an `{ error: ...}` type here but not elsewhere?

Reply:
ah! this slipped though my PR. I reverted it back. The next commit has a plan for my a better fix for this case.